pkg/libovsdb/schema.go

```diff
@@ -1,29 +1,37 @@
 package libovsdb
 
 import (
+	"encoding/json"
 	"fmt"
 	"io"
+	"strings"
 )
 
+type Schema map[string]*DatabaseSchema
```

Comment:
why do you need this `Shema` type. if you want to store several schemas in the server you can define it there. 
RFC7047 Schema definition is exactly `DataBaseSchema`

Reply:
renamed to `Schemas`